src/parse_data.py

```diff
@@ -0,0 +1,27 @@
+import json
+import logging
+from typing import Generator, Union, List, Dict, Any
+from datetime import datetime
+
+logger = logging.getLogger('parseStoriesInfo')
+
+def build_stories_info(stories_generator: Generator[dict, None, None]) -> str:
+    logger.info('Parse stories data')
+    stories_info: Dict[str, Any] = dict()
+
+    now = datetime.now()    
+    stories_info['date'] = str(now)
```

Comment:
If I see 'date' within an entity data structure I would think that's a 'business process' related date, not some meta-information about the retrieval time.

Reply:
Just get it from template that was discussed in Slack. I will rename to `retrieval_time`